requirements.mac.txt

```diff
@@ -1,185 +1,18 @@
-#
-# This file is manually created from the autogenerated 
-# requirements.txt . It is experimental to support MAC
-# (intel, apple silicon and gpus via mps). For this we
-# currently need the nightly torch and torchvision.
-#
---extra-index-url https://download.pytorch.org/whl/nightly/cpu
+h5py>=3.7
+lapy>=1.0.1
+matplotlib>=3.7.1
+nibabel>=5.1.0
+numpy>=1.25,<2
+pandas>=1.5.3
+pyyaml>=6.0
+scikit-image>=0.19.3
+scikit-learn>=1.2.2
+scipy>=1.10.1,!=1.13.0
+simpleitk>=2.2.1
+tensorboard>=2.12.1
+torch>=2.0.1+cu117
+torchio>=0.18.83
+torchvision>=0.15.2+cu117
```

Comment:
The cuXXX designations should not work on mac. I think you need to remove that.

Reply:
it worked, but I removed it anyway. thanks.